### DISCUSSION TITLE
Welcome to OnlyT Discussions!

### DISCUSSION THREAD

**Discussion by AntonyCorbett** [AntonyCorbett/OnlyT#435] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. If you have a question or an idea, this is probably the place to start. Please create a new discussion post rather than replying to this one!

**Comment by Gia-P** [comment]
Hello Antony 

How are you? I hop you are well 
I have one idea 
Is it possible to add chairman's comment time (1 min) counter after students done there talk? 

EX:  
    1. Bible Reading: (4 min.) Jos 3:1-13 (th study 10) >> Student finished 
    2. chairman >> starts talking Thank you so much ... and Modulation (th study 10) HOW TO DO IT:
    Same time (1 min) counter is running

**Reply by AntonyCorbett** [reply]
OnlyT deals with this automatically. Whenever counsel is to be offered, after the student timer is stopped, the display shows how long the student talk took. This static display remains on-screen for a little over 1 minute (or however long you specify in the settings) so that the chairman can refer to it if needed. When the minute is up, the timer display reverts to the standard clock - giving the chairman notice that his 1 minute is up. It's simply a matter of training the chairman to understand these cues.

**Reply by MiltJR111** [reply]
> OnlyT deals with this automatically. Whenever counsel is to be offered, after the student timer is stopped, the display shows how long the student talk took. This static display remains on-screen for a little over 1 minute (or however long you specify in the settings) so that the chairman can refer to it if needed. When the minute is up, the timer display reverts to the standard clock - giving the chairman notice that his 1 minute is up. It's simply a matter of training the chairman to understand these cues.

Does this apply to the web clock too? It seems like that display feature doesn't apply, and that's how our congregation is using the timer over Zoom. Can that feature also be added to the web clock?

**Reply by AntonyCorbett** [reply]
@MiltJR111 Thanks for your question. No, that feature is not part of the web clock. I've locked the conversation as this is the "Welcome" post. As mentioned above, please create a new discussion post rather than replying to this one. In this case there are several posts on this subject in the "Issues" pages. Thanks.
